saleor/core/prices.py

```diff
@@ -5,11 +5,18 @@
 from babel.numbers import get_currency_precision
 from prices import Money, TaxedMoney, TaxedMoneyRange
 
+from saleor import settings
+
 if TYPE_CHECKING:
     from django.db.models import Model
 
 PriceType = TypeVar("PriceType", TaxedMoney, Money, Decimal, TaxedMoneyRange)
 
+# The maximum price value we can save in the database
+MAXIMUM_PRICE = (
+    10 ** (settings.DEFAULT_MAX_DIGITS - settings.DEFAULT_DECIMAL_PLACES) - 1
+)
```

Comment:
If this depends on these settings, maybe `MAXIMUM_PRICE` should be also moved to settings, close to `DEFAULT_MAX_DIGITS` and `DEFAULT_DECIMAL_PLACES`? Not sure if we ever going to change these values, but if we did, we might not notice that MAXIUM_PRICE depends on them.

Reply:
Why should we care about MAXIMUM_PRICE dependencies?